sqlmesh/core/model/definition.py

```diff
@@ -1888,6 +1925,20 @@ def _python_env(
                         used_macros[name] = macros[name]
                     elif name in variables:
                         used_variables.add(name)
+                elif (
+                    macro_func_or_var.__class__ is exp.Identifier and "@" in macro_func_or_var.this
```

Comment:
isinstance(macro_func_or_var, exp.Identifier)

Reply:
Keeping it consistent with the other 2 branches. Guess I should change all of them.